docs/markdown/DevGuide_InviwoPy.md

```diff
@@ -0,0 +1,72 @@
+# Using Inviwo from within Python
+While Python Processors (TODO: link) allow you to embed Python code inside Inviwo processors, you can also use Inviwopy to use your Inviwo networks from within a Python application. This guide shows you first how to build Inviwopy, then how to load existing Inviwo workspaces for use in your Python scripts and lastly some ways to modify a loaded Inviwo network from within Python.
+
+## Building InviwoPy
+To use Inviwopy, you have to build the appropriate `.dll`/`.so` yourself, since it is currently not available through Pypi. To do so, enable the `IVW_MODULE_PYTHON3` and `IVW_MODULE_PYTHON3QT` CMake flags. Next you need to specify the Python executable to which the produced library shall be compatible in the `PYTHON_EXECUTABLE` flag.
+<details>
+<summary>
+Using Inviwopy with Anaconda environments
+</summary>
+<p>
+
+1. Set the `PYTHON_EXECUTABLE` flag to your environment's executable (e.g. `~/.conda/envs/inviwo/bin/python3`)
+
+2. If not set automatically, also adapt the `PYTHON_LIBRARY` flag to `<conda env>/lib/libpython3.6m.so` (according to your Python version).
+
+</p>
+</details>
+
+ Depending on your Python version and operating system this will generate two dynamic libraries with names similar to:
+- `inviwopy.cpython-36m-x76_64-linux-gnu.so`
+- `inviwopyapp.cpython-36m-x76_64-linux-gnu.so`
```

Comment:
just to notet these dll depends on most other dll in the bin folder of the build too...